LCLSGeneral/LCLSGeneral/Data types/EPICS/ST_EpicsMotorMSTA.TcDUT

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <DUT Name="ST_EpicsMotorMSTA" Id="{8dc1ba89-a69d-4583-94a8-23394c1f2a52}">
+    <Declaration><![CDATA[TYPE ST_EpicsMotorMSTA :
```

Comment:
Confirmed this struct matches https://epics.anl.gov/bcda/synApps/motor/motorRecord.html#Fields_status